app/src/main/res/layout-land/admin_auth_activity.xml

```diff
@@ -102,12 +76,25 @@
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toTopOf="parent" />
 
-            <org.oppia.app.profile.ProfileInputView
+          <TextView
+            android:id="@+id/admin_auth_sub_text"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="12dp"
+            android:fontFamily="sans-serif"
+            android:text="@string/admin_auth_sub"
```

Comment:
Please correct subtext as per the mock.

Reply:
Actually the subtext and toolbar title are  dynamic based on which option you are selecting: 
## Add Profile:
![Screenshot_1585549982](https://user-images.githubusercontent.com/9396084/77882480-a5b88300-727e-11ea-9c91-680e623fe3e4.png)


## Admin Controls:
![Screenshot_1585549986](https://user-images.githubusercontent.com/9396084/77882487-a81add00-727e-11ea-85cb-4808944dbc05.png)


This has been finalised with @mschanteltc in one of the team meetings.